.github/workflows/automatic_pull_request_review.yml

```diff
@@ -0,0 +1,82 @@
+name: Automatic pull request review
+
+on:
+  pull_request:
+    types: [opened, reopened, synchronize]
```

Comment:
Should we execute this workflow on `edited` type, i.e. PR title changes to remove WIP :thinking:

Reply:
@luongvo nice spot there, since there are conditions to check for PR content changes as well like the WIP check. Will update 👍